.github/workflows/pr-review.yml

```
@@ -68,8 +68,14 @@ jobs:
 
       - name: Upload coverage to Codecov
         uses: codecov/codecov-action@v4
+<<<<<<< HEAD
```

actionlint (1.7.10)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)